### DISCUSSION TITLE
Can access Git Kraken and other feature for the student pack , except copilot

### DISCUSSION THREAD

**Discussion by Ehab98** [community/community#134852] [GitHub Education]
### Hi everyone,

This message keep apear 
You don't have access to GitHub Copilot.
![Screenshot from 2024-08-06 12-01-44](https://github.com/user-attachments/assets/7822ffab-e437-4133-a517-462eb80358d1)
![Screenshot from 2024-08-06 12-03-09](https://github.com/user-attachments/assets/e89ac4bc-fdf1-480b-b12b-2dff0a4887e8)
![Screenshot from 2024-08-06 12-03-59](https://github.com/user-attachments/assets/41277567-d18e-44c4-b5eb-0c88279074e1)

**Comment by davevad93** [comment]
Hi @Ehab98,

From the screenshots you have provided it looks like you haven't currently logged in to `Copilot`, try the following:

1. Log in with your current GitHub account.
2. Copy the 8-digit verification code to the clipboard.
3. Paste it into the `Device Activation` pop-up window.

By following those steps you should be able to use `Copilot`, if you still can't, you should contact support by opening a ticket [here](https://support.github.com/contact/education), select 
`I'm a verified student and I'm having trouble redeeming my Global Campus offers.` and explain your issue.
